lib/batch/batch_metadata_updates_634.rb

```diff
@@ -0,0 +1,73 @@
+# batch updates for https://github.com/galterlibrary/digital-repository/issues/634
+
+# Updates number 1:
+#   - All items with "Pediatric Neurology Briefs Publishers" in the Original 
+#   - Publisher field --> Add additional field to Digital Publisher with "Pediatric 
+#   - Neurology Briefs Publishers" and delete from Original Publisher
+
+pnb_publishers = "Pediatric Neurology Briefs Publishers"
+items_needing_original_publisher_updates = GenericFile.where("original_publisher" => pnb_publishers) + 
+                                           Collection.where("original_publisher" => pnb_publishers)
+
+items_needing_original_publisher_updates.each do |item|
+  item.publisher += [pnb_publishers]
+  item.original_publisher -= [pnb_publishers]
+  item.save
+end
+
+
+# Updates number 2:
+#   - All items in Science in Society Collection - delete "Science in Society" 
+#   - from Original Publisher 
+
+sis_publisher = "Science in Society"
+
+def update_sis_original_publisher(id)
+  collection = Collection.find(id)
+
+  if !collection.members.empty?
+    collection.members.each do |member|
+      if member.class == Collection 
+        update_sis_original_publisher(member.id)
+      else
+        member.original_publisher -= [sis_publisher]
+        member.save
+      end
+    end
+  end
+
+  collection.original_publisher -= [sis_publisher]
+  collection.save!
+end
+
+sis_collection = "93c81706-47f8-49a1-88a2-3acbc971f4ed"
+update_sis_original_publisher(sis_collection)
+
+
+# Updates number 3:
+#   - All DigitalHub items - delete "Chicago, Illinois, United States" 
+#   - and delete "Chicago, IL, USA" from Location
+
+chicago_locations = ["Chicago, Illinois, United States", "Chicago, IL, USA"]
+items_needing_based_near_updates = GenericFile.where("based_near" => chicago_locations) +
```

Comment:
What does `=>` in `"based_near" => chicago_locations` mean ? Does it mean contains "one or the other" or "both"? I am guessing it means "one or the other".

Reply:
I did check this, and yes it returns "one or the other".